scaffolding-chef-inspec/tests/user-linux-default/inspec.yml

```diff
@@ -0,0 +1,8 @@
+name: user-linux-default
+title: Effortless Audit with default plan settings
+maintainer: CCP Team
+copyright: CCP Team
+copyright_email: you@example.com
+license: Apache-2.0
+summary: Effortless Audit Linux Baseline
+version: 0.1.0
```

Comment:
Please add `pkg_upstream_url="https://inspec.io"`

Reply:
this is part of testing I think these changes are unnecessary as they don't affect the visibility of the tests.

Reply:
Including the pkg_upstream_url is part of the required metadata for plans we use, but I understand if for a test you think it's unnecessary, so we can ignore this one. :)